pkg/network/node/node.go

```diff
@@ -405,6 +363,53 @@ func (node *OsdnNode) Start() error {
 `), 0644)
 }
 
+// reattachPods takes an array containing the information about pods that had been
+// attached to the OVS bridge before restart, and either reattaches or kills each of the
+// corresponding pods.
+func (node *OsdnNode) reattachPods(existingPods map[string]podNetworkInfo) error {
+	sandboxes, err := node.getPodSandboxes()
+	if err != nil {
+		return err
+	}
+
+	failed := []*kruntimeapi.PodSandbox{}
+	for sandboxID, podInfo := range existingPods {
+		sandbox, ok := sandboxes[sandboxID]
+		if !ok {
+			glog.Warningf("Could not find sandbox for existing pod with IP %s; it may be in an inconsistent state", podInfo.ip)
```

Comment:
So this is the case where the ovsdb has a sandbox that isn't in the CRI, right? Why wouldn't we just want to delete the entry from the ovsdb?

Reply:
We already did delete it from the ovsdb; `existingPods` is the record of what *used to be* in the db before we destroyed and recreated the bridge.